app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(10, 3)
+    assert sum(result) == 10, (f"Expected sum "
+                               f"to be 10, but got {sum(result)}")
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(12, 3)
```

Comment:
The assignment of `result` is unnecessary for this simple test. You can directly use the function call in the `assert` statement as per the checklist recommendation.